styles/globals.css

```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
How do we use this for all text?

Reply:
I added new code. The default font is now Inter, you can always change the default font by uncommenting the fontFamily code in tailwind.config.js and global.css files